--- gnome-pim-1.4.0/gnomecard/Makefile.am~	Sat Nov 24 07:19:35 2001
+++ gnome-pim-1.4.0/gnomecard/Makefile.am	Sat Nov 24 07:19:35 2001
@@ -65,7 +65,7 @@
 	gnome-pim-stubs.c	\
 	gnome-pim.h
 
-CLEANFILES += $(CORBA_SRCLIST)
+CLEANFILES = $(CORBA_SRCLIST)
 
 $(CORBA_SRCLIST): $(top_srcdir)/idl/gnome-pim.idl $(ORBIT_IDL)
 	orbit-idl $(top_srcdir)/idl/gnome-pim.idl
--- gnome-pim-1.4.0/gncal/Makefile.am~	Sat Nov 24 07:55:51 2001
+++ gnome-pim-1.4.0/gncal/Makefile.am	Sat Nov 24 07:55:51 2001
@@ -6,8 +6,6 @@
 gnorbadir = $(sysconfdir)/CORBA/servers
 gnorba_DATA = gnomecal.gnorba
 
-help_base = $(datadir)/gnome/help/cal
-
 if HAVE_GNOME_PILOT
 extra_pilot_bins = 				\
 		calendar-conduit-control-applet	\
