diff -urN gnome-pim-1.4.0.org/Makefile.am gnome-pim-1.4.0/Makefile.am
--- gnome-pim-1.4.0.org/Makefile.am	Thu May 17 17:31:52 2001
+++ gnome-pim-1.4.0/Makefile.am	Thu May 17 17:32:25 2001
@@ -1,4 +1,5 @@
 SUBDIRS = po idl macros \
+	 intl \
 	 libversit gncal gnomecard
 
 EXTRA_DIST = gnome-pim.spec.in gnome-pim.keys.in gnome-pim.spec
diff -urN gnome-pim-1.4.0.org/configure.in gnome-pim-1.4.0/configure.in
--- gnome-pim-1.4.0.org/configure.in	Thu May 17 17:31:52 2001
+++ gnome-pim-1.4.0/configure.in	Thu May 17 17:34:25 2001
@@ -105,7 +105,7 @@
 
 ALL_LINGUAS="az ca cs cy da de el en_GB es et fi fr ga gd gl gv hu it ja ko kw lt nl no pl pt pt_BR ru sk sl sv tr uk wa zh_CN.GB2312 zh_TW.Big5"
 dnl AM_GNU_GETTEXT
-AM_GNOME_GETTEXT
+AM_GNU_GETTEXT
 
 AC_SUBST(CAPPLET_LIBS)
 AC_SUBST(CFLAGS)
@@ -116,6 +116,7 @@
 gnome-pim.spec
 Makefile
 po/Makefile.in
+intl/Makefile
 idl/Makefile
 macros/Makefile
 gncal/Makefile
