diff -uNr gnome-pim-1.4.0.orig/gnomecard/dialog.c gnome-pim-1.4.0/gnomecard/dialog.c
--- gnome-pim-1.4.0.orig/gnomecard/dialog.c	Thu Jun 13 22:02:28 2002
+++ gnome-pim-1.4.0/gnomecard/dialog.c	Thu Jun 13 22:01:42 2002
@@ -1122,12 +1122,12 @@
 
 static void gnomecard_send_read_error (char *fname)
 {
-	gnomecard_send_error (fname, "reading");
+	gnomecard_send_error (fname, _("reading"));
 }
 
 static void gnomecard_send_write_error (char *fname)
 {
-	gnomecard_send_error (fname, "writing");
+	gnomecard_send_error (fname, _("writing"));
 }
 
 static void gnomecard_append_call(GtkWidget *widget, gpointer data)
--- gnome-pim-1.4.6/po/pl.po.orig	Thu Apr 25 21:53:28 2002
+++ gnome-pim-1.4.6/po/pl.po	Fri Jun 14 17:29:37 2002
@@ -15,20 +15,20 @@
 #: gncal/calendar-conduit.c:920 gncal/calendar-conduit.c:972
 #: gncal/calendar-conduit.c:1405
 msgid "Error while communicating with calendar server"
-msgstr ""
+msgstr "B³±d podczas komunikacji z serwerem kalendarza."
 
 #: gncal/calendar-conduit.c:620
 #, c-format
 msgid "Calendar holds %d entries"
-msgstr ""
+msgstr "Kalendarz zawiera %d wpisów"
 
 #: gncal/calendar-conduit.c:642 gncal/calendar-conduit.c:644
 msgid "Could not start gnomecal server"
-msgstr ""
+msgstr "Nie mo¿na uruchomiæ serwera gnomecal"
 
 #: gncal/calendar-conduit.c:672 gncal/calendar-conduit.c:675
 msgid "Could not read pilot's DateBook application block"
-msgstr ""
+msgstr "Nie uda³o sie odczytaæ bloku aplikacji z DateBook pilota"
 
 #. This array must be in the same order as enumerations
 #. in GnomePilotConduitSyncType as they are used as index.
@@ -37,32 +37,32 @@
 #: gncal/calendar-conduit-control-applet.c:53
 #: gnomecard/address-conduit-capplet.c:67
 msgid "Disabled"
-msgstr ""
+msgstr "Wy³±czone"
 
 #: gncal/calendar-conduit-control-applet.c:54
 #: gnomecard/address-conduit-capplet.c:68
 msgid "Synchronize"
-msgstr ""
+msgstr "Synchronizuj"
 
 #: gncal/calendar-conduit-control-applet.c:55
 #: gnomecard/address-conduit-capplet.c:69
 msgid "Copy From Pilot"
-msgstr ""
+msgstr "Kopiuj z Pilota"
 
 #: gncal/calendar-conduit-control-applet.c:56
 #: gnomecard/address-conduit-capplet.c:70
 msgid "Copy To Pilot"
-msgstr ""
+msgstr "Kopiuj do Pilota"
 
 #: gncal/calendar-conduit-control-applet.c:57
 #: gnomecard/address-conduit-capplet.c:71
 msgid "Merge From Pilot"
-msgstr ""
+msgstr "Do³±cz z Pilota"
 
 #: gncal/calendar-conduit-control-applet.c:58
 #: gnomecard/address-conduit-capplet.c:72
 msgid "Merge To Pilot"
-msgstr ""
+msgstr "Do³±cz do pilota"
 
 #: gncal/calendar-conduit-control-applet.c:147
 #, fuzzy
@@ -71,21 +71,21 @@
 
 #: gncal/calendar-conduit-control-applet.c:148
 msgid "(C) 1998"
-msgstr ""
+msgstr "(C) 1998"
 
 #: gncal/calendar-conduit-control-applet.c:150
 msgid "Configuration utility for the calendar conduit.\n"
-msgstr ""
+msgstr "Konfigurator ³±cza do kalendarza.\n"
 
 #: gncal/calendar-conduit-control-applet.c:191
 #: gnomecard/address-conduit-capplet.c:201
 msgid "Synchronize Action"
-msgstr ""
+msgstr "Synchronizuj operacjê"
 
 #: gncal/calendar-conduit-control-applet.c:262
 #: gncal/todo-conduit-control-applet.c:194
 msgid "Conduit state"
-msgstr ""
+msgstr "Stan ³±cza"
 
 #: gncal/calendar-conduit-control-applet.c:309
 #: gncal/calendar-conduit-control-applet.c:324
@@ -97,11 +97,13 @@
 "No pilot configured, please choose the\n"
 "'Pilot Link Properties' capplet first."
 msgstr ""
+"Nie skonfigurowano Pilota, wybierz przedtem\n"
+"modu³ konfiguracyjny 'Pilot Link Properties'."
 
 #: gncal/calendar-conduit-control-applet.c:330
 #: gnomecard/address-conduit-capplet.c:418
 msgid "Not connected to the gnome-pilot daemon"
-msgstr ""
+msgstr "Brak po³±czenia z serwerem gnome-pilot"
 
 #: gncal/calendar-conduit-control-applet.c:335
 #: gnomecard/address-conduit-capplet.c:423
@@ -109,6 +111,8 @@
 "An error occured when trying to fetch\n"
 "pilot list from the gnome-pilot daemon"
 msgstr ""
+"Wyst±pi³ b³±d podczas pobierania listy\n"
+"pilota z serwera gnome-pilot"
 
 #: gncal/calendar-conduit-control-applet.c:364
 #: gnomecard/address-conduit-capplet.c:449
@@ -116,83 +120,87 @@
 "Could not create object for conduit.\n"
 "Perhaps the conduit isn't properly installed ?"
 msgstr ""
+"Nie mo¿na utworzyæ obiektu dla ³±cza.\n"
+"Byæ mo¿e, ³±cze nie zosta³o poprawnie zainstalowane?"
 
 #: gncal/calendar-pilot-sync.c:60
 msgid "Specifies the port on which the Pilot is"
-msgstr ""
+msgstr "Podaj port na na którym jest Polot"
 
 #: gncal/calendar-pilot-sync.c:60
 msgid "PORT"
-msgstr ""
+msgstr "PORT"
 
 #: gncal/calendar-pilot-sync.c:62
 msgid "If you want to debug the attributes on records"
-msgstr ""
+msgstr "li chcesz ¶ledziæ attrybuty rekordów"
 
 #: gncal/calendar-pilot-sync.c:64
 msgid "Only syncs from desktop to pilot"
-msgstr ""
+msgstr "Synchronizuje jedynie z pulpitu do pilota"
 
 #: gncal/calendar-pilot-sync.c:66
 msgid "Only syncs from pilot to desktop"
-msgstr ""
+msgstr "Synchronizuje jedynie z pilota do pulpitu"
 
 #: gncal/calendar-pilot-sync.c:89
 msgid "Can not create Pilot socket\n"
-msgstr ""
+msgstr "Nie mo¿na utworzyæ gniazda Pilota\n"
 
 #: gncal/calendar-pilot-sync.c:96
 #, c-format
 msgid "Can not bind to device %s\n"
-msgstr ""
+msgstr "Nie mo¿na dowi±zaæ do urz±dzenia %s\n"
 
 #: gncal/calendar-pilot-sync.c:99
 msgid "Failed to get a connection from the Pilot device"
-msgstr ""
+msgstr "Nie uda³o siê otrzymaæ po³±czenia od urz±dzania Pilota"
 
 #: gncal/calendar-pilot-sync.c:102
 msgid "pi_accept failed"
-msgstr ""
+msgstr "pi_accept siê nie powiod³o"
 
 #: gncal/calendar-pilot-sync.c:167
 msgid "\tObject did not exist, creating a new one\n"
-msgstr ""
+msgstr "\tObiekt nie istnia³, tworzony jest nowy\n"
 
 #: gncal/calendar-pilot-sync.c:174
 msgid ""
 "\tObject has been modified on desktop and on the pilot, desktop takes "
 "precedence\n"
 msgstr ""
+"\tObiekt zosta³ zmodyfikowany na pulpicie i w pilocie; pulpit ma "
+"pierwszeñstwo\n"
 
 #: gncal/calendar-pilot-sync.c:475
 msgid "No description"
-msgstr ""
+msgstr "Brak opisu"
 
 #: gncal/calendar-pilot-sync.c:547
 msgid "Syncing with the pilot..."
-msgstr ""
+msgstr "Synchronizacja z pilotem..."
 
 #: gncal/calendar-pilot-sync.c:554
 msgid "Could not open DatebookDB on the Pilot"
-msgstr ""
+msgstr "Nie ud±³o siê otworzyæ DatebookDB z pilota"
 
 #: gncal/calendar-pilot-sync.c:555
 msgid "Unable to open DatebookDB"
-msgstr ""
+msgstr "Nie uda³o siê otworzyæ DatebookDB"
 
 #: gncal/calendar-pilot-sync.c:617
 msgid "Synced DateBook from Pilot to GnomeCal"
-msgstr ""
+msgstr "Zsynchronizowano DateBook z Pilota do GnomeCal"
 
 #: gncal/calendar.c:429
 #, c-format
 msgid "Error while creating backup file: %s"
-msgstr ""
+msgstr "B³±d podczas tworzenia kopii zapasowej pliku: %s"
 
 #: gncal/calendar.c:452 gncal/calendar.c:456
 #, c-format
 msgid "Error writing the calendar file: %s"
-msgstr ""
+msgstr "B³±d podczas zapisu pliku kalendarza: %s"
 
 #: gncal/calobj.c:999
 msgid "Appointment"
@@ -277,7 +285,7 @@
 
 #: gncal/eventedit.c:767
 msgid "An event's end time must be later than its start time."
-msgstr ""
+msgstr "Czas koñca zdarzenia musi byæ pó¼niejszy ni¿ czas jego pocz±tku."
 
 #: gncal/eventedit.c:855
 msgid "General"
@@ -885,16 +893,16 @@
 
 #: gncal/gnome-cal.c:472
 msgid "Snooze"
-msgstr ""
+msgstr "Drzemka"
 
 #. Idea: we need Snooze option :-)
 #: gncal/gnome-cal.c:484 gncal/gnome-cal.c:529
 msgid "Ok"
-msgstr ""
+msgstr "Ok"
 
 #: gncal/gnome-month-item.c:15
 msgid "-*-helvetica-medium-r-normal--10-*-*-*-p-*-*-*"
-msgstr ""
+msgstr "-*-helvetica-medium-r-normal--10-*-*-*-p-*-*-2"
 
 #: gncal/goto.c:82
 msgid "Year:"
@@ -954,15 +962,15 @@
 
 #: gncal/main.c:70
 msgid "To-Do item that is not yet due:"
-msgstr ""
+msgstr "Pozycja nie jest jeszcze 'do zrobienia':"
 
 #: gncal/main.c:71
 msgid "To-Do item that is due today:"
-msgstr ""
+msgstr "Pozycji 'do zrobienia' przypada na dzi¶:"
 
 #: gncal/main.c:72
 msgid "To-Do item that is overdue:"
-msgstr ""
+msgstr "Pozycja 'do zrobienia' jest zaleg³a:"
 
 #: gncal/main.c:283
 msgid "Gnome Calendar"
@@ -974,7 +982,7 @@
 
 #: gncal/main.c:463
 msgid "File not found"
-msgstr ""
+msgstr "Nie znaleziono pliku"
 
 #: gncal/main.c:482
 msgid "Open calendar"
@@ -1120,11 +1128,11 @@
 
 #: gncal/main.c:920
 msgid "Enable debugging output of TYPE (alarms)"
-msgstr ""
+msgstr "W³±cza komunikaty ¶ledzenia dla TYPU (alarmów)"
 
 #: gncal/main.c:920
 msgid "TYPE"
-msgstr ""
+msgstr "TYP"
 
 #: gncal/modify-cats.c:63
 #, fuzzy
@@ -1138,94 +1146,94 @@
 
 #: gncal/modify-cats.c:67
 msgid "Please select categories for the current item"
-msgstr ""
+msgstr "Wybierz kategorie dla bie¿±cego wpisu"
 
 #: gncal/modify-cats.c:71
 #, fuzzy
 msgid "Selected"
-msgstr "Usuñ"
+msgstr "Wybrane"
 
 #: gncal/modify-cats.c:72
 msgid "Available"
-msgstr ""
+msgstr "Dostêpne"
 
 #: gncal/modify-cats.c:98
 msgid "Add a category"
-msgstr ""
+msgstr "Dodaj kategoriê"
 
 #: gncal/modify-cats.c:99
 msgid "Add a category to the list of selected categories"
-msgstr ""
+msgstr "Dodaj kategoriê do listy wybranych kategorii"
 
 #: gncal/modify-cats.c:102
 msgid "Remove a category"
-msgstr ""
+msgstr "Usuñ kategoriê"
 
 #: gncal/modify-cats.c:103
 msgid "Remove a category from  the list of selected categories"
-msgstr ""
+msgstr "Usuñ kategoriê z listy wybranych kategorii"
 
 #: gncal/modify-cats.c:410
 #, fuzzy
 msgid "Church"
-msgstr "godzina"
+msgstr "Ko¶ció³"
 
 #: gncal/modify-cats.c:411
 #, fuzzy
 msgid "Friends"
-msgstr "Znajd¼"
+msgstr "Przyjaciele"
 
 #: gncal/modify-cats.c:412
 msgid "Guadec"
-msgstr ""
+msgstr "Guadec"
 
 #: gncal/modify-cats.c:413
 #, fuzzy
 msgid "Hacking"
-msgstr "Nag³ówki:"
+msgstr "Hacking"
 
 #: gncal/modify-cats.c:414
 msgid "Hobby"
-msgstr ""
+msgstr "Hobby"
 
 #: gncal/modify-cats.c:415
 msgid "Housekeeping"
-msgstr ""
+msgstr "Prace domowe"
 
 #: gncal/modify-cats.c:416
 #, fuzzy
 msgid "Love"
-msgstr "Usuñ"
+msgstr "Mi³o¶æ"
 
 #: gncal/modify-cats.c:417
 msgid "Meeting"
-msgstr ""
+msgstr "Spotkanie"
 
 #: gncal/modify-cats.c:418
 #, fuzzy
 msgid "Money"
-msgstr "Pon"
+msgstr "Pieni±dze"
 
 #: gncal/modify-cats.c:419
 msgid "Music"
-msgstr ""
+msgstr "Muzyka"
 
 #: gncal/modify-cats.c:420
 #, fuzzy
 msgid "Paperwork"
-msgstr "Nowy"
+msgstr "Prace biurowe"
 
 #: gncal/modify-cats.c:421
 msgid "Relatives"
-msgstr ""
+msgstr "Krewni"
 
 #: gncal/modify-cats.c:422
 msgid "School"
-msgstr ""
+msgstr "Szko³a"
 
 #: gncal/modify-cats.c:423
 msgid "Shopping"
-msgstr ""
+msgstr "Zakupy"
 
 #: gncal/modify-cats.c:424
 #, fuzzy
@@ -1487,10 +1495,355 @@
 msgid "Address Conduit Options"
 msgstr ""
 
+#: gnomecard/address-conduit-capplet.c:215
+msgid "Vadim Strizhevsky <vadim@optonline.net>"
+msgstr "Vadim Strizhevsky <vadim@optonline.net>"
+
+#: gnomecard/address-conduit-capplet.c:217
+msgid "Gpilotd address conduit"
+msgstr "£±cze adresowe do gpilotd"
+
+#: gnomecard/address-conduit-capplet.c:218
+msgid "(C) 1998 the Free Software Foundation"
+msgstr "(C) 1998 the Free Software Foundation"
+
+#: gnomecard/address-conduit-capplet.c:220
+msgid ""
+"Configuration utility for the address conduit.\n"
+"The address conduit syncronizes addresses from the pilot with GnomeCard file "
+"on the local host"
+msgstr ""
+"Konfigurator ³±cza adresowego.\n"
+"£±cze adresowe synchronizuje adresy pilota z plikiem GnomeCard na lokalnym "
+"hoscie."
+
+#: gnomecard/address-conduit-capplet.c:223
+msgid "gnome-unknown.xpm"
+msgstr "gnome-unknown.xpm"
+
+#: gnomecard/address-conduit-capplet.c:324
+msgid "GnomeCard File:"
+msgstr "Plik GnomeCard:"
+
+#: gnomecard/address-conduit-capplet.c:402
+msgid "Address Conduit State"
+msgstr "Stan ³±cza adresowego"
+
+#: gnomecard/address-conduit-capplet.c:408
+msgid "Address Conduit Options"
+msgstr "Opcje ³±cza adresowego"
+
+#: gnomecard/address-conduit.c:403 gnomecard/address-conduit.c:405
+#, c-format
+msgid "Unable to open file: %s"
+msgstr "Nie mo¿na otworzyæ pliku: %s"
+
+#: gnomecard/address-conduit.c:914 gnomecard/address-conduit.c:916
+msgid "No filename specified. Please run address conduit capplet first."
+msgstr ""
+"Nie podano nazwy pliku. Uruchom przedtem modu³ konfiguracji ³±cza adresowego."
+
 #: gnomecard/canvas.c:164
-#, fuzzy
 msgid "No Card Name."
-msgstr "Nazwa karty"
+msgstr "Karta bez nazwy."
+
+#: gnomecard/card.c:1676
+msgid "Card: "
+msgstr "Karta: "
+
+#: gnomecard/card.c:1678
+msgid ""
+"\n"
+"Name: "
+msgstr ""
+"\n"
+"Imiê i nazwisko: "
+
+#: gnomecard/card.c:1679
+msgid ""
+"\n"
+"  Prefix:     "
+msgstr ""
+"\n"
+"  Prefiks:     "
+
+#: gnomecard/card.c:1680
+msgid ""
+"\n"
+"  Given:      "
+msgstr ""
+"\n"
+"  Imiê:        "
+
+#: gnomecard/card.c:1681
+msgid ""
+"\n"
+"  Additional: "
+msgstr ""
+"\n"
+"  Drugie imiê: "
+
+#: gnomecard/card.c:1682
+msgid ""
+"\n"
+"  Family:     "
+msgstr ""
+"\n"
+"  Nazwisko:    "
+
+#: gnomecard/card.c:1683
+msgid ""
+"\n"
+"  Suffix:     "
+msgstr ""
+"\n"
+"  Sufiks:      "
+
+#: gnomecard/card.c:1697
+msgid ""
+"\n"
+"Birth Date: "
+msgstr ""
+"\n"
+"Data urodzenia: "
+
+#: gnomecard/card.c:1708
+msgid ""
+"\n"
+"Address:"
+msgstr ""
+"\n"
+"Adres:"
+
+#: gnomecard/card.c:1710
+msgid ""
+"\n"
+"  Postal Box:  "
+msgstr ""
+"\n"
+"  Skr. pocztowa: "
+
+#: gnomecard/card.c:1711
+msgid ""
+"\n"
+"  Ext:         "
+msgstr ""
+
+#: gnomecard/card.c:1712
+msgid ""
+"\n"
+"  Street:      "
+msgstr ""
+"\n"
+"  Ulica:        "
+
+#: gnomecard/card.c:1713
+msgid ""
+"\n"
+"  City:        "
+msgstr ""
+"\n"
+"  Miasto:       "
+
+#: gnomecard/card.c:1714
+msgid ""
+"\n"
+"  Region:      "
+msgstr ""
+"\n"
+"  Region:       "
+
+#: gnomecard/card.c:1715
+msgid ""
+"\n"
+"  Postal Code: "
+msgstr ""
+"\n"
+"  Kod pocztowy: "
+
+#: gnomecard/card.c:1716
+msgid ""
+"\n"
+"  Country:     "
+msgstr ""
+"\n"
+"  Kraj:         "
+
+#: gnomecard/card.c:1729
+msgid ""
+"\n"
+"Delivery Label: "
+msgstr ""
+
+#: gnomecard/card.c:1741
+msgid ""
+"\n"
+"Telephones:\n"
+msgstr ""
+"\n"
+"Telefony:\n"
+
+#: gnomecard/card.c:1744
+msgid ""
+"\n"
+"Telephone:"
+msgstr ""
+"\n"
+"Telefon:"
+
+#: gnomecard/card.c:1768
+msgid ""
+"\n"
+"E-mail:\n"
+msgstr ""
+"\n"
+"E-mail:\n"
+
+#: gnomecard/card.c:1771
+msgid ""
+"\n"
+"E-mail:"
+msgstr ""
+"\n"
+"E-mail:"
+
+#: gnomecard/card.c:1790
+msgid ""
+"\n"
+"Mailer: "
+msgstr ""
+"\n"
+"Czytnik poczty: "
+
+#: gnomecard/card.c:1796
+msgid ""
+"\n"
+"Time Zone: "
+msgstr ""
+"\n"
+"Strefa czasowa: "
+
+#: gnomecard/card.c:1804
+msgid ""
+"\n"
+"Geo Location: "
+msgstr ""
+"\n"
+"Po³o¿enie geogr.: "
+
+#: gnomecard/card.c:1808
+msgid ""
+"\n"
+"Title: "
+msgstr ""
+"\n"
+"Tytu³: "
+
+#: gnomecard/card.c:1809
+msgid ""
+"\n"
+"Business Role: "
+msgstr ""
+"\n"
+"Stanowisko: "
+
+#: gnomecard/card.c:1821
+msgid ""
+"\n"
+"Org: "
+msgstr ""
+"\n"
+"Organizacja: "
+
+#: gnomecard/card.c:1822
+msgid ""
+"\n"
+"  Name:  "
+msgstr ""
+"\n"
+"  Nazwa:      "
+
+#: gnomecard/card.c:1823
+msgid ""
+"\n"
+"  Unit:  "
+msgstr ""
+"\n"
+"  Jednostka:  "
+
+#: gnomecard/card.c:1824
+msgid ""
+"\n"
+"  Unit2: "
+msgstr ""
+"\n"
+"  Jednostka2: "
+
+#: gnomecard/card.c:1825
+msgid ""
+"\n"
+"  Unit3: "
+msgstr ""
+"\n"
+"  Jednostka3: "
+
+#: gnomecard/card.c:1826
+msgid ""
+"\n"
+"  Unit4: "
+msgstr ""
+"\n"
+"  Jednostka4: "
+
+#: gnomecard/card.c:1830
+msgid ""
+"\n"
+"Categories: "
+msgstr ""
+"\n"
+"Kategorie: "
+
+#: gnomecard/card.c:1831
+msgid ""
+"\n"
+"Comment: "
+msgstr ""
+"\n"
+"Komentarz: "
+
+#. if (crd->sound.prop.used) {
+#. if (crd->sound.type != SOUND_PHONETIC)
+#. addPropSizedValue(string, _("\nPronunciation: "),
+#. crd->sound.data, crd->sound.size);
+#. else
+#. add_strProp_to_string(string, _("\nPronunciation: "),
+#. crd->sound.data);
+#.
+#. add_SoundType(string, crd->sound.type);
+#. }
+#: gnomecard/card.c:1844
+msgid ""
+"\n"
+"URL: "
+msgstr ""
+"\n"
+"URL: "
+
+#: gnomecard/card.c:1845
+msgid ""
+"\n"
+"Unique String: "
+msgstr ""
+"\n"
+"Identyfikator: "
+
+#: gnomecard/card.c:1848
+msgid ""
+"\n"
+"Public Key: "
+msgstr ""
+"\n"
+"Klucz publiczny: "
 
 #: gnomecard/card.c:1677
 #, fuzzy
@@ -1793,16 +2146,15 @@
 
 #: gnomecard/deladdrlist.c:30
 msgid "Post Office:"
-msgstr ""
+msgstr "Poczta"
 
 #: gnomecard/deladdrlist.c:30
 msgid "Extended:"
 msgstr ""
 
 #: gnomecard/deladdrlist.c:30
-#, fuzzy
 msgid "Street:"
-msgstr "Ulica (1)"
+msgstr "Ulica:"
 
 #: gnomecard/deladdrlist.c:31
 msgid "City:"
@@ -1810,12 +2162,11 @@
 
 #: gnomecard/deladdrlist.c:31
 msgid "Region:"
-msgstr ""
+msgstr "Region:"
 
 #: gnomecard/deladdrlist.c:31
-#, fuzzy
 msgid "Postal Code:"
-msgstr "Skrzynka pocztowa"
+msgstr "Kod pocztowy:"
 
 #: gnomecard/deladdrlist.c:32
 msgid "Country:"
@@ -1882,7 +2233,7 @@
 #. create name frame
 #: gnomecard/dialog.c:339 gnomecard/tree.c:153
 msgid "Name"
-msgstr "Nazwisko"
+msgstr "Dane osoby"
 
 #. first name
 #: gnomecard/dialog.c:347
@@ -1914,18 +2265,16 @@
 msgstr "Tytu³:"
 
 #: gnomecard/dialog.c:439
-#, fuzzy
 msgid "Network"
-msgstr "Nowy"
+msgstr "Sieæ"
 
 #: gnomecard/dialog.c:445
 msgid "Addresses"
 msgstr "Adresy"
 
 #: gnomecard/dialog.c:462
-#, fuzzy
 msgid "Birthdate:"
-msgstr "Stan:"
+msgstr "Data urodzenia:"
 
 #: gnomecard/dialog.c:487
 msgid "Geographical"
@@ -1947,9 +2296,10 @@
 msgid "Geographic Position"
 msgstr "Po³o¿enie geograficzne"
 
+# the space is okay, but the comma is superflous -ke-
 #: gnomecard/dialog.c:525
 msgid "lat, "
-msgstr "szer."
+msgstr "szer. "
 
 #: gnomecard/dialog.c:533
 msgid "lon."
@@ -2356,7 +2706,7 @@
 
 #: gnomecard/phonelist.c:32
 msgid "Bulletin Board"
-msgstr "Tablica og³oszeniowa"
+msgstr "Tablica / skrzynka og³oszeniowa"
 
 #: gnomecard/phonelist.c:33
 msgid "Modem"
