--- gnome-pim-1.4.9/gncal/Makefile.am	Fri Dec 20 16:39:04 2002
+++ gnome-pim-1.4.9.n/gncal/Makefile.am	Mon Dec 15 18:56:50 2003
@@ -53,18 +53,19 @@
 	$(GNOME_LIBDIR) \
 	$(GNOMEGNORBA_LIBS) \
 	$(INTLLIBS) \
+	GnomeCal-common.lo \
+	alarm.lo \
+	calendar.lo \
+	calobj.lo \
+	timeutil.lo \
 	$(top_builddir)/libversit/libversit.a
 
 gnomecal_SOURCES = \
-	GnomeCal-common.c	\
 	GnomeCal-skels.c	\
 	GnomeCal.h		\
-	alarm.c			\
 	alarm.h			\
-	calendar.c		\
 	calendar.h		\
 	calendar-conduit.h	\
-	calobj.c		\
 	calobj.h		\
 	eventedit.c		\
 	eventedit.h		\
@@ -103,7 +104,6 @@
 	prop.c			\
 	quick-view.c		\
 	quick-view.h		\
-	timeutil.c		\
 	timeutil.h		\
 	todo-conduit.h		\
 	view-utils.h		\
--- gnome-pim-1.4.9/gnomecard/Makefile.am	Tue Mar 12 01:41:58 2002
+++ gnome-pim-1.4.9.n/gnomecard/Makefile.am	Mon Dec 15 18:58:37 2003
@@ -14,7 +14,6 @@
 
 gnomecard_SOURCES =	\
         $(CORBA_SRCLIST) \
-	card.c		\
 	card.h		\
 	columnhdrs.c    \
 	columnhdrs.h    \
@@ -39,7 +38,6 @@
 	list.h		\
 	misc.c          \
 	misc.h          \
-	my.c            \
 	my.h            \
 	phonelist.c     \
 	phonelist.h     \
@@ -72,6 +73,8 @@
 
 gnomecard_LDADD = $(LINK_FLAGS)\
 	$(GNOME_LIBDIR) \
+	card.lo \
+	my.lo \
 	$(GNOMEGNORBA_LIBS)
 
 if HAVE_GNOME_PILOT
