diff -Nru gnome-pim-1.4.0/gnomecard/help/C/Makefile.am gnome-pim-1.4.0/gnomecard/help.new/C/Makefile.am
--- gnome-pim-1.4.0/gnomecard/help/C/Makefile.am	Thu Jan  1 01:00:00 1970
+++ gnome-pim-1.4.0/gnomecard/help.new/C/Makefile.am	Mon May  3 08:51:13 1999
@@ -0,0 +1,33 @@
+gnomecard_helpdir = $(datadir)/gnome/help/gnomecard/C
+
+gnomecard_help_DATA = 	\
+	index.html		\
+	topic.dat
+
+EXTRA_DIST = gnomecard.sgml topic.dat
+
+all: index.html
+
+index.html: gnomecard/index.html
+	-cp $(srcdir)/gnomecard/index.html .
+
+gnomecard/index.html: gnomecard.sgml
+	-(cd $(srcdir); db2html gnomecard.sgml) > /dev/null
+
+
+dist-hook:
+	mkdir $(distdir)/gnomecard
+	-cp $(srcdir)/gnomecard/*.html $(distdir)/gnomecard
+	-cp $(srcdir)/gnomecard/*.css $(distdir)/gnomecard
+
+install-data-local: index.html
+	-for file in $(srcdir)/gnomecard/*.html $(srcdir)/gnomecard/*.css; do \
+	  basefile=`echo $$file | sed -e 's,^.*/,,'`; \
+	  $(INSTALL_DATA) $$file $(DESTDIR)$(gnomecard_helpdir)/$$basefile; \
+	done
+
+gnomecard.ps: gnomecard.sgml
+	-(cd $(srcdir); db2ps $<) > /dev/null
+
+gnomecard.rtf: gnomecard.sgml
+	-(cd $(srcdir); db2rtf $<) > /dev/null
diff -Nru gnome-pim-1.4.0/gnomecard/help/C/gnomecard.sgml gnome-pim-1.4.0/gnomecard/help.new/C/gnomecard.sgml
--- gnome-pim-1.4.0/gnomecard/help/C/gnomecard.sgml	Thu Jan  1 01:00:00 1970
+++ gnome-pim-1.4.0/gnomecard/help.new/C/gnomecard.sgml	Mon Feb  1 00:40:03 1999
@@ -0,0 +1,248 @@
+<!DOCTYPE BOOK PUBLIC "-//Davenport//DTD DocBook V3.0//EN" [
+<!ENTITY GNOME-APPS SYSTEM "gnomecard.sgml">] >
+
+
+<Book id="index"><title>GNOME Address Book users guide.</title>
+ <bookinfo>
+ <authorgroup>
+   <author>
+    <firstname>Arturo</firstname><surname>Espinosa Aldama</surname>
+   </author>
+  </authorgroup>
+  <releaseinfo>documentation in progress</releaseinfo>
+  <legalnotice>
+   <para>
+    This document can be freely redistributed according to the
+    terms of the GNU General Public License.
+   </para>
+  </legalnotice>
+ </bookinfo>
+ <toc></toc>
+ 
+ <chapter id="description"><title>Description</title>
+  <para>
+   GnomeCard is the combination of an addressbook (contact manager) and an
+   electronic business card manager. It saves and loads files using
+   the <ulink URL="http://www.imc.org/pdi">vCard</ulink> format and aims
+   to implement every aspect of this proposal.
+  </para>
+ </chapter>
+   
+ <chapter id="Display"><title>The Display</title>
+   <para>
+    At the moment, the program shows two views, a menu bar and a toolbar.
+   </para>
+  <sect1 id="listview"><title>The List View</title>
+   <para>
+    From the list view, the user can visualize every contact registered in
+    the loaded file, with the possibility to configure the fields that will
+    be shown from the records. There is always a selected record that is
+    called the "current record", whose fields are shown in the <link
+    linkend="recordview">record view</link>.
+   </para>
+   <sect2 id="listviewactions"><title>List View Actions</title>
+    <para>
+     The list view is used to perform the actions of selection, sorting and
+     DnD.
+    </para>
+    <sect3 id="lvaselection"><title>List View Selection</title>
+     <para>
+      To select a card as the currently viewed card, scroll the list view
+      until the desired record is visible and click on it. This will make
+      the <link linkend="recordview">record view</link> to show the selected 
+      record's information.
+     </para>
+    </sect3>
+    <sect3 id="lvasorting"><title>List View Sorting</title>
+     <para>
+      Perform a sort on the card list by clicking on the title of the
+      column by which the list needs to be sorted.
+     </para>
+    </sect3>
+    <sect3 id="lvadnd"><title>List View Drag and Drop</title>
+     <para>
+      It is possible to drag a record and drop it in other applications,
+      like mail clients, calendars and other addressbooks. (not implemented,
+      yet).
+     </para>
+    </sect3>
+   </sect2>
+  </sect1>
+  <sect1 id="recordview"><title>The Record View</title>
+   <para>
+    From the record view, the user is able to view all the fields associated
+    with the current record selected on the list view, including photos and
+    logos that may come with the card.
+   </para>
+  </sect1>
+  <sect1 id="menubar"><title>The Menu Bar</title>
+   <para>
+    All possible actions in GnomeCard, except for DnD, have an equivlent
+    inside the menu hierarchy of the Menu Bar. Here is the Menu Bar tree:
+    <itemizedlist>
+     <listitem override="File">
+      <para>Operation over files and exiting.</para>
+      <itemizedlist>
+       <listitem override="New">
+        <para>Create a completly new addressbook.</para>
+       </listitem>
+       <listitem override="Open...">
+        <para>Open an addressbook file.</para>
+       </listitem>
+       <listitem override="Save">
+        <para>Save the current addressbook.</para>
+       </listitem>
+       <listitem override="Save As...">
+        <para>Save the current addressbook with a different name.</para>
+       </listitem>
+       <listitem override="Merge with...">
+        <para>Merge contents of a file with the current data.</para>
+       </listitem>
+       <listitem override="Exit">
+        <para>Close GnomeCard.</para>
+       </listitem>
+      </itemizedlist>
+     </listitem>
+     <listitem override="Edit">
+      <para>Actions concerning the edition of the addressbook</para>
+      <itemizedlist>
+       <listitem override="Card">
+        <para>Opens the Edit dialog to change the contents of the current
+	record.</para>
+       </listitem>
+       <listitem override="Delete">
+        <para>Erases the current card.
+	 <warning><title>Warning</title>
+	  <para>
+	   The erased card won't be able to be recovered and there are
+	   no confirmations.
+	  </para>
+	 </warning>
+	</para>
+       </listitem>
+       <listitem override="Go">
+        <para>Changes the selected card.</para>
+       </listitem>
+       <listitem override="Find">
+        <para>
+	 Make a pattern search in every field of every record. You
+         will be presented with the <link linkend="dialogfind">Find Dialog
+	 </link>.
+	</para>
+       </listitem>
+       <listitem override="Sort">
+        <para>Sort the list using different fields as the criteria.</para>
+       </listitem>
+      </itemizedlist>
+     </listitem>
+     <listitem override="Settings">
+      <para>Options for changing the way GnomeCard shows the information
+       or behaves.
+      </para>
+      <itemizedlist>
+       <listitem override="Preferences...">
+        <para>Bring up the <link linkend="dialogpref">Preferences
+         Dialog</link>, which sets up the layout of the list view and other
+         settings that cannot be represented in the Settings menu.
+	</para>
+       </listitem>
+      </itemizedlist>
+     </listitem>
+    </itemizedlist>
+   </para>
+  </sect1>
+  <sect1 id="toolbar"><title>The Toolbar</title>
+   <para>
+    The toolbar presents the most usual actions in a more straight-away
+    fashion than the menu bar. Clicking any option from the toolbar results
+    the same as selecting its menu equivalent.
+   </para>
+  </sect1>
+  <sect1 id="dialogs"><title>Dialogs</title>
+   <para>
+    This section explains the behaviour of most of the dialogs that come up
+    in GnomeCard. Those that are not documented here are either messages or
+    "standard", like the Save As and Open dialogs.
+   </para>
+   <sect2 id="dialogedit"><title>Edit Dialog</title>
+    <para>
+     You will get to this dialog when selecting "Edit Card" or "Add Card"
+     from the toolbar, or Edit/Add Card or Edit/Card options from the menu bar.
+    </para>
+    <para>
+     This is the largest dialog of all, and the one that is used to add or
+     modify data from the existing or new records in the addressbook. It is
+     divided in several sections which cover the different types of data
+     that a card can have. If the dialog came up to edit the data from an
+     existing card, the used fields will be filled up with the card's data.
+     You may add or modify any of the information presented. Otherwise, the
+     dialog will come up empty, ready to receive all the information to be
+     added.
+    </para>
+   </sect2>
+   <sect2 id="dialogpref"><title>Preferences Dialog</title>
+    <para>
+     Some of the most complicated settings are presented here for the user
+     to modify.
+    </para>
+    <sect3 id="dialogpreflay"><title>Layout Section</title>
+     <para>
+      Customize the columns to be shown by the List View through this
+      interfase. At the left, the possible columns that GnomeCard can
+      present on the list are shown. At the right, the columns that
+      the List View currently shows can be seen.
+     </para>
+     <para>
+      Select a Possible Column from the left and press 
+      <guibutton>Add</guibutton> to pass it to the right. If the Column is 
+      already in the Displayed Columns set, <guibutton>Add</guibutton> will 
+      be grayed out.
+     </para>
+     <para>
+      To remove a Displayed Column from the list, select it on the right and
+      press <guibutton>Remove</guibutton>.
+     </para>
+     <para>
+      Selecting a Displayed Column and pressing <guibutton>Move Up</guibutton>
+      or <guibutton>Move Down</guibutton>. A Column will be displayed to the
+      leftmost in the List View if it is on the top of the Displayed Columns
+      list.
+     </para>
+    </sect3>
+   </sect2>
+   <sect2 id="dialogfind"><title>Find Dialog</title>
+    <para>
+     The Find Dialog will make a pattern search starting from the current
+     record on every field of every card and set the current record to the
+     first one that matches the criteria. If the lookup reaches the top or
+     the bottom of the list, GnomeCard will ask if the search should be 
+     continued.
+    </para>
+    <para>
+     Pattern matching is done using the typical shell wildcards. Check 
+     <ulink URL="man:bash(1)#lbBC">bash (1)</ulink> under the "Pathname 
+     Expansion" subsection for a complete reference.
+    </para>
+    <para>
+     Case sensitiveness and direction of the search. If case sensitiveness
+     is set on, "Rivas" won't match "rivas".
+    </para>
+   </sect2>
+  </sect1>
+ </chapter>
+</book>
+
+<!-- Keep this comment at the end of the file
+Local variables:
+mode: sgml
+sgml-omittag:t
+sgml-shorttag:t
+sgml-minimize-attributes:nil
+sgml-always-quote-attributes:t
+sgml-indent-step:2
+sgml-indent-data:t
+sgml-exposed-tags:nil
+sgml-local-catalogs:nil
+sgml-local-ecat-files:nil
+End:
+-->
diff -Nru gnome-pim-1.4.0/gnomecard/help/C/topic.dat gnome-pim-1.4.0/gnomecard/help.new/C/topic.dat
--- gnome-pim-1.4.0/gnomecard/help/C/topic.dat	Thu Jan  1 01:00:00 1970
+++ gnome-pim-1.4.0/gnomecard/help.new/C/topic.dat	Mon Feb  1 00:40:03 1999
@@ -0,0 +1 @@
+index.html GNOME Address Book users guide.
diff -Nru gnome-pim-1.4.0/gnomecard/help/Makefile.am gnome-pim-1.4.0/gnomecard/help.new/Makefile.am
--- gnome-pim-1.4.0/gnomecard/help/Makefile.am	Thu Jan  1 01:00:00 1970
+++ gnome-pim-1.4.0/gnomecard/help.new/Makefile.am	Mon Feb  1 00:40:02 1999
@@ -0,0 +1 @@
+SUBDIRS = C
--- gnome-pim-1.4.0/gnomecard/Makefile.am~	Sat Nov 24 07:46:15 2001
+++ gnome-pim-1.4.0/gnomecard/Makefile.am	Sat Nov 24 07:46:15 2001
@@ -1,4 +1,4 @@
-nSUBDIRS = help
+SUBDIRS = help
 
 xpmhelp_base = $(datadir)/gnome/help/cal
 
