diff -ur gnome-pim-1.4.9.old/gncal/Makefile.am gnome-pim-1.4.9/gncal/Makefile.am
--- gnome-pim-1.4.9.old/gncal/Makefile.am	Tue Dec 16 13:02:32 2003
+++ gnome-pim-1.4.9/gncal/Makefile.am	Tue Dec 16 13:38:46 2003
@@ -58,7 +58,7 @@
 	calendar.lo \
 	calobj.lo \
 	timeutil.lo \
-	$(top_builddir)/libversit/libversit.a
+	$(top_builddir)/libversit/libversit_lt.la
 
 gnomecal_SOURCES = \
 	GnomeCal-skels.c	\
Only in gnome-pim-1.4.9/gncal: Makefile.am~
diff -ur gnome-pim-1.4.9.old/gnomecard/Makefile.am gnome-pim-1.4.9/gnomecard/Makefile.am
--- gnome-pim-1.4.9.old/gnomecard/Makefile.am	Tue Dec 16 13:02:32 2003
+++ gnome-pim-1.4.9/gnomecard/Makefile.am	Tue Dec 16 13:38:32 2003
@@ -66,7 +66,7 @@
 
 LINK_FLAGS = \
 	$(INTLLIBS)	\
-	$(top_builddir)/libversit/libversit.a
+	$(top_builddir)/libversit/libversit_lt.la
 
 gnomecard_LDADD = $(LINK_FLAGS)\
 	$(GNOME_LIBDIR) \
Only in gnome-pim-1.4.9/gnomecard: Makefile.am~
diff -ur gnome-pim-1.4.9.old/libversit/Makefile.am gnome-pim-1.4.9/libversit/Makefile.am
--- gnome-pim-1.4.9.old/libversit/Makefile.am	Thu Jun 27 20:25:32 2002
+++ gnome-pim-1.4.9/libversit/Makefile.am	Tue Dec 16 13:37:46 2003
@@ -11,38 +11,10 @@
 	vcaltmp.c	\
 	vcaltmp.h
 
-noinst_LIBRARIES = libversit.a
 noinst_LTLIBRARIES = libversit_lt.la
 
-libversit_a_SOURCES = $(VERSIT_SRC)
 libversit_lt_la_SOURCES = $(VERSIT_SRC)
 
 EXTRA_DIST = README.TXT vcaltest.c vctest.c
 
 YFLAGS=-pversit_
-
-
-
-# Left to its own devices, make will do the following with the
-# above declarations:
-#   1. vcc.y -> vcc.c
-#   2. vcc.c -> vcc.o
-#   3. vobject.c -> vobject.o
-#   4. vcaltmp.c -> vcaltmp.o
-#   5. *.o -> libversit.a
-#   6. vcc.c -> vcc.lo vcc.o
-#   7. vobject.c -> vobject.lo vobject.o
-#   8. vcaltmp.c -> vcaltmp.lo vcaltmp.o
-#   9. *.lo -> libversit_lt.la
-#
-# This has the annoying side effect of leaving libversit.a out of date
-# with respect to its dependencies (since the .o files were rebuilt in
-# steps 6-8), causing it to get rebuilt again the next time make comes
-# through, causing additional re-linking further down the line.
-#
-# This hacks around that. (Badly, since all it ends up doing is moving
-# step 5 to the end, so you still end up compiling each of the files
-# an extra time...)
-
-libversit_a_DEPENDENCIES = libversit_lt.la
-
Only in gnome-pim-1.4.9/libversit: Makefile.am~
